// Try ve Catch Kullanımı

let items;

try{
    items.forEach(item => console.log(item));
} catch (error){
    console.log("Error! : ",error)
    //items = [1,2,3]
    //items.forEach(item => console.log(item));
}

console.log("hata yönetimi düzgün çalışıyor!")


let info = "Kodluyoruz"
console.log(info)





